### ISSUE TITLE
Option to retrieve notes on classification through `getKlass()` function.

### ISSUE THREAD

**Issue by mafw** [statisticsnorway/klassR#13]
To me it would be helpful if I could pass an argument to the `getKlass()` function that makes it also fetch a column with notes on the classification.  I find that notes are helpful for the aggregation codes. For example, the code FGK7 from the [Codelist for Accounting functions](https://www.ssb.no/klass/klassifikasjoner/277/koder) has (in Norwegian) an informative note that lets me know what codes are included in the aggregation. I can retrieve this information by using the `GET()` function directly:
```
resp <- httr::GET("https://data.ssb.no/api/klass/v1/classifications/277/codes.json?from=2015-01-01&selectCodes=FGK7") 
content <- httr::content(resp, as  = "text")
code <- jsonlite::fromJSON(content, flatten = TRUE)$codes
code$notes
```
It would, however, be neat if the `getKlass()` function had an option to return this information directly.  Maybe through a logical option named "notes"? 
```
getKlass(277, notes = T)
```
Alternatively the function could return the notes column by default if any of the classifications has a note?

If you think this sounds reasonable, I can try to implement this functionality.

**Comment by sjentoft** [comment]
Thanks for your idea! I think this sounds good so please come with a code suggestion if you can. Otherwise I will look into it for the next version.

**Comment by sjentoft** [comment]
Hi! I have now included this option in klassR version 0.2.0. Thanks for the idea.

**Comment by mafw** [comment]
Great, thank you!
